### ISSUE TITLE
Change the full screen page color scheme

### ISSUE THREAD

**Issue by DNin01** [TurboWarp/extensions#983]
It feels much more fitting for the full screen background to blend in with the colors of the project. Like, it's great when you have a dark background around a dark game. To help with this, I think it would be cool if the background color in full screen view could be changed.

**Comment by snowboyz0825** [comment]
I'm not am expert at all, but if I had to guess that probably can't be done becasue it has to do with the browser being used or the turbowarp player itself. Personally, I wished I could do the same thing, so I coded a kind-of workaround

![block_9_12_2023-7_02_16 PM](https://github.com/TurboWarp/extensions/assets/101284660/ca6e21e4-1728-4f47-9a70-f814080a862b)

If you want to download the file, here it is
https://github.com/snowboyz0825/snowscodestuff/blob/main/Full%20Screen%20Code.sprite3

It will leave a border around it if you're fullscreen in the turbowarp editor, but you should be able to change this with the add on for it. 

In any packaged project, it should make it fullscreen. If you'd like the stage to stay the same size, i'm not entirely sure how to help you there. You'd have to code it in yourself, or if you want to send me the specific project file, I could give it a shot. Anyways, hope this helps

**Comment by DNin01** [comment]
Yeah, it's worth thinking about how this should work outside turbowarp.org.

**Comment by snowboyz0825** [comment]
> Yeah, it's worth thinking about how this should work outside turbowarp.org.

The code snippet or the extension?

**Comment by DNin01** [comment]
> > Yeah, it's worth thinking about how this should work outside turbowarp.org.
> 
> 
> 
> The code snippet or the extension?

The extension in general.

**Comment by snowboyz0825** [comment]
> > > Yeah, it's worth thinking about how this should work outside turbowarp.org.
> > 
> > 
> > The code snippet or the extension?
> 
> The extension in general.

If you package it as an EXE file, it should automatically resize itself to match the window. If it's an HTML file, it should extend it up to the tab-bar thingy at the top of your browser. For the other options, I'm not entirely sure what it'd do

**Comment by milkyestway** [comment]
I mean if you enable browser fullscreen in addons with project controls as visible on hover, it doesn’t even have a border so, problem solved? A few extensions have browser fullscreen on/off as option and f11 also works I think.
